R/initialisers.R

```diff
@@ -0,0 +1,108 @@
+#' initialise_analytics
+#'
```

Comment:
Worth giving a bit more of a description or at least pointing to the package readme for usage details?